src/Symfony/Bridge/Doctrine/Test/DoctrineTestHelper.php

```diff
@@ -14,6 +14,7 @@
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
 use Doctrine\ORM\EntityManager;
+use PHPUnit\Framework\TestCase;
```

Comment:
We actually don't want 5.4.3, as the namespace was not the same (lowercase vs not lowercase). Use the first version using the ``PHPUnit`` case instead.